server/main.py

```diff
@@ -94,3 +133,32 @@ async def calculate_spectrum(payload: Payload):
                 "units": spectrum.units[payload.mode],
             },
         }
+
+
+# "/download-spectrum"--> to return back the .spec file and delete that file after giving the fileresponse back
+@app.post("/download-spectrum")
+async def download_spec(payload: Payload, background_tasks: BackgroundTasks):
+
+    try:
+        make_file()
```

Comment:
This function doesn't make a file so I would call it something else, maybe  `create_download_directory()`?